lib/features/login_screen/presentation/view/widgets/custom_login_appbar.dart

```diff
@@ -7,7 +8,7 @@ class CustomLoginAppbar extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-                    height: 42,
+                    height: 42.h,
```

Comment:
### use **sh** by screen util to dived home screen between three widgets (gradient container, heart image , list items).
 **example :** 

- _gradient container_ height : .15.sh.
- _heart image_ height : .30.sh.
- _list items_ height : 55.sh
@KhalidEl-MariAa